agent/src/main/java/io/atomix/agent/AtomixAgent.java

```diff
@@ -99,6 +99,18 @@ public File convert(ArgumentParser argumentParser, Argument argument, String val
         .required(false)
         .setDefault(new File(System.getProperty("user.dir"), "data"))
         .help("The server data directory");
+    parser.addArgument("--core-partitions", "-cp")
+        .type(Integer.class)
+        .metavar("NUM")
+        .required(false)
+        .setDefault(7)
+        .help("The number of core partitions");
+    parser.addArgument("--data-partitions", "-dp")
+        .type(Integer.class)
+        .metavar("NUM")
+        .required(false)
+        .setDefault(71)
```

Comment:
Is this a reasonable default?

Reply:
Just the same default as elsewhere. Probably needs to be reconsidered separately